services/m3dbnode/config/config_test.go

```diff
@@ -370,6 +370,7 @@ bootstrap:
   fs:
     numProcessorsPerCPU: 0.125
   peers: null
+  cacheSeriesMetadata: null
```

Comment:
Want to also update the example config?  `./example/m3db-node-config.yaml` I think it is